### ISSUE TITLE
ap0110 doesn't respring into jailbroken mode even after reboot, but it does do something that tricks regular h3lix into believing it's already jailbroken

### ISSUE THREAD

**Issue by maemolol** [athenusdev/ap0110#4]
btw i have an iPad3,5

**Comment by spv420** [comment]
yeah this project is essentially dead, autoexecution was an interesting concept but didn't have a ton of practical use for jailbreaks. p0laris eta son tho

if you're not familiar, i'm working solo on a proper untethered jailbreak for 9.x and 10.x on 32-bit: p0laris.
